test/test_torch.py

```diff
@@ -11548,7 +11596,7 @@ def test_vander(self, device):
             torch.vander(torch.stack((x, x)))
 
         # This passes on the xla backend
-        if device != 'xla':
+        if device != 'xla' and device != 'cpu':
```

Comment:
`self.device_type != 'xla' and self.device_type != 'cpu':`

Reply:
Done.